doc/hostname.md

```diff
@@ -0,0 +1,33 @@
+## Hostname in Linux ##
+
+We have several hostnames in linux. Every of them has different usage and different place where to set it up. Some of them can be configured in the YaST's lan module.
+
+## Local system hostname / static hostname (/etc/hostname) ##
+
+You can put a string into /etc/hostname. This string is than used as so called "local system hostname" which is set in the kernel during the boot. This name is used for identification of your local system. This name is typically used e.g. in logs where individual log entries can be prepended by this name. More importantly this name is not related to networking, so putting e.g. FQDN (Fully Qualified Domain Name) has no sense.
+
+Content of this file can be displayed by ```hostname``` command or in systemd world by ```hostnamectl --static```
+
+In YaST you can edit this hostname on Hostname / DNS tab, hostname field.
```

Comment:
Why about adding a screenshot? And taking in account that neither the kernel nor hostnamectl have validation or restriction about using a FQDN how YaST handle that situation?